sw/device/lib/handler.c

```diff
@@ -13,31 +14,16 @@
  */
 static uint32_t get_mtval(void) {
   uint32_t mtval;
-  asm volatile("csrr %0, mtval" : "=r"(mtval) : :);
+  asm volatile("csrr %0, mtval" : "=r"(mtval));
   return mtval;
 }
 
-/**
- * Default Error Handling
- * @param error message supplied by caller
- * TODO - this will be soon by a real print formatting
- */
-static void print_exc_msg(const char *msg) {
-  const uint32_t mtval = get_mtval();
-  uart_send_str((char *)msg);
-  uart_send_str("MTVAL value is ");
-  uart_send_uint(mtval, 32);
-  uart_send_str("\n");
-  while (1) {
-  };
-}
-
 // Below functions are default weak exception handlers meant to be overriden
 __attribute__((weak)) void handler_exception(void) {
   uint32_t mcause;
   exc_id_t exc_cause;
 
-  asm volatile("csrr %0 , mcause" : "=r"(mcause) : :);
+  asm volatile("csrr %0 , mcause" : "=r"(mcause));
```

Comment:
Same commit as the other asm change.  Also insert space between `"=r"` and `(mcause)`.